vnext/Shared/HermesRuntimeHolder.cpp

```diff
@@ -419,6 +419,12 @@ bool HermesJSRuntime::supportsConsole() const {
   return false;
 }
 
+std::unique_ptr<facebook::react::jsinspector_modern::StackTrace> HermesJSRuntime::captureStackTrace(
```

Comment:
Does this impact our debugging story?

Reply:
It shouldn't impact the current debugging story since this is an augmentation. https://github.com/facebook/react-native/commit/4fbc1f2ef8bef00d6577e44fbbe505dca3eba691

I've opened #13260 to track adding an actual implementation. Things are a bit tricky at the moment since we forked JSExecutor during the last integration. #13172 is tracking that.